packages/pytest-simcore/src/pytest_simcore/environment_configs.py

```diff
@@ -38,8 +38,19 @@ def all_env_devel_undefined(
     delenvs_from_dict(monkeypatch, env_devel_dict, raising=False)
 
 
+def pytest_addoption(parser: pytest.Parser):
```

Comment:
so this works even though this is not a conftests file? nice. I guess that is because these are in pytest-simcore.

Reply:
@sanderegg yes, provided that the plugin `pytest_simcore.environment_configs` is added in the list, otherwise the option does not appear in `pytest --help` under the sexction `pytest-simcore options`
SE 
![image](https://github.com/ITISFoundation/osparc-simcore/assets/32402063/9a8370cb-aacc-4e22-8c03-2bef8e24a7a6)